### PR TITLE
NAV-211 Read new workflow state structure.

### PR DESCRIPTION
**Pull request by jonathansberry**
Updates Engine to read the new workflow state structure.

### CODE REVIEW

File: navigator_engine/tests/unit_tests/test_conditionals.py [hunk 1]
```diff
@@ -32,9 +32,9 @@ def test_check_manual_confirmation(mock_engine, action_id, expected):
         'navigator-workflow-state': {
             'data': {
                 'completedTasks': [
-                    "task1",
-                    "task2",
-                    "task3"
+                    {"id": "task1", "completedAt": "2021-12-10 15:22:12.640480"},
```
Comment (tomeksabala) [inline navigator_engine/tests/unit_tests/test_conditionals.py:35]: API uses different date format: `"Mon, 13 Dec 2021 16:42:47 GMT"`

**Review by tomeksabala — APPROVED** [review]
There's an error in test data but I think it can be fixed later when the task complete invalidate feature is in place.

File: navigator_engine/tests/unit_tests/test_conditionals.py (hunk 1 above)
Comment (jonathansberry) [reply navigator_engine/tests/unit_tests/test_conditionals.py:35]: I copied this straight from your own API tests

Comment (jonathansberry) [reply navigator_engine/tests/unit_tests/test_conditionals.py:35]: I can update with the main NAV-211 PR

**Pull request merged**